playbooks/ansible-network-iosxr-appliance/files/bootstrap.yaml

```diff
@@ -21,15 +21,30 @@
   gather_facts: false
   connection: local
   tasks:
+
+    - set_fact:
+        _iosxr_password: "admin"
+
+    - set_fact:
+        _iosxr_password: "admin123"
+      when: item in ["iosxr-7.0.2"]
+      with_inventory_hostnames: appliance
+
+    # - name: Remove ssh key generated by Automatic Host Key Generation
+    #  shell: "ssh-keygen -R {{ hostvars[item].nodepool.private_ipv4 }}"
+    #  args:
+    #    executable: /bin/bash
+    #  with_inventory_hostnames: appliance
```

Comment:
okay, lets add this back for now. Adding a NOTE here to explain what we think is happening.